include/zephyr/drivers/dai.h

```diff
@@ -200,6 +200,7 @@ struct dai_properties {
  * @param options Dai specific configuration options.
  * @param word_size Number of bits representing one data word.
  * @param block_size Size of one RX/TX memory block (buffer) in bytes.
+ * @param link_config Link Hda configuration.
```

Comment:
Ack, @jxstelter I get that part but this struct is a generic struct for all implementations of dais in Zephyr (from all vendors). We already have field like "options" described as "dai specific configuration options". It would seem "link_config" is really is semantically a similar field. Let's try to find a way to add this in a way that can be used by other types of dais.